src/databases/repositories/vipRepository.ts

```diff
@@ -1,2 +1,36 @@
+import { HashedUserID } from "../../types/user.model";
+import { db } from "../databases";
+import { IDatabase } from "../IDatabase";
+import { PrivateTempVipLog } from "../models/private/tempVipLog";
+
 export class VipRepository {
+    public static async addTempVip(adminUserId: string, userId: string, privateDb: IDatabase): Promise<void> {
+        const tempVipLog = new PrivateTempVipLog({
+            issuerUserID: adminUserId,
+            targetUserID: userId,
+            enabled: true,
+            updatedAt: Date.now()
+        });
+
+        await privateDb.prepare(
+            "run",
+            `INSERT INTO "tempVipLog" VALUES (?, ?, ?, ?)`,
+            [tempVipLog.issuerUserID, tempVipLog.targetUserID, + tempVipLog.enabled, tempVipLog.updatedAt]
+        );
```

Comment:
Why create this object to then just split it back out into an array?

Reply:
What type of modification do you mean? I more mean why not just pass in the values directly into the query instead of indirectly through the class